### ISSUE TITLE
Question: Tree view & table view planned?

### ISSUE THREAD

**Issue by jlgerber** [redox-os/orbtk#135]
I see a list view, and a ticket covering a TabsWidget, but i dont see any plans for a treeview or a table view. Are these planned? They seem to be pretty standard fare in ui toolkits (qt, gtk, etc)

**Comment by FloVanGH** [comment]
TreeView and TableView are to important widgets and will be part of OrbTk. Not all issues are not yet moved from gitlab. I hope I could finish it in January.

**Comment by kivimango** [comment]
Yes, its planned, I did some very early draft for TableView, but it is the far most complex widget with sorting columns adding rows, etc and I need(ed) some experience with orbtk before I start planning and coding.
